backend/tracim_backend/tests/functional/test_simple_search.py

```diff
@@ -160,6 +160,135 @@ def test_api___simple_search_ok__by_filename(
         assert search_result["is_total_hits_accurate"] is False
         assert search_result["contents"][0]["label"] == first_content_name
 
+    @parameterized.expand(
+        [
+            # content_name, search_string, nb_content_result, first_content_name, content_body
+            # exact syntax
+            ("good practices", "texttosearch", 1, "good practices", "texttosearch"),
+            ("good practices", "search", 1, "good practices", "texttosearch"),
+        ]
+    )
+    def test_api___simple_search_ok__by_content(
+        self, content_name, search_string, nb_content_result, first_content_name, content_body
```

Comment:
same remark about naming and order of parameters

Reply:
done.